porting/building-ubports-boot.rst

```diff
@@ -63,15 +59,15 @@ With the file open, change the ``src``, or first attribute, of the ``/data`` mou
 
 Now, remove all ``context=`` options from all block devices in the file. Save and exit.
 
-For an example of this, see `this commit on universalsuperbox/android_device_motorola_potter <https://github.com/UniversalSuperBox/android_device_motorola_potter/commit/9b574967e3a6f07884760b418befe731ccfcb924>`_. Removing the ``wait`` flag is not required and was an accident.
+For an example of this, see `this commit on universalsuperbox/android_device_motorola_potter <https://github.com/UniversalSuperBox/android_device_motorola_potter/commit/9b574967e3a6f07884760b418befe731ccfcb924>`__. Removing the ``wait`` flag is not required and was an accident.
```

Comment:
why double underscores?

Reply:
I've decided to start using double underscores when I'm sure that I'll not need the target again. Keeps the link namespace cleaner, less chance of name collisions. http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases

Reply:
ok